cypress/integration/test_brewing_process.js

```diff
@@ -0,0 +1,46 @@
+describe('Testing of recipe brewing operations', () => {
+  it('Starts recipe', () => {
+    // go to /recipe site from default baseUrl in cypress.json
+    cy.visit('recipe');
+    cy.get('.chamber:nth-child(2)').click();
```

Comment:
naco tam je toto? neni to zbytocne

Reply:
chcel som aby sa preklikol druhy recept asi nestastne sa vola chamber classa toho buttonu, ale zatial to staci aj takto si myslim